client/lib/domains/utils/get-domain-type.js

```diff
@@ -12,7 +12,7 @@ export function getDomainType( domainFromApi ) {
 		return domainTypes.TRANSFER;
 	}
 
-	if ( domainFromApi.wpcom_domain ) {
+	if ( domainFromApi.wpcom_domain || domainFromApi.type === 'wpcom' ) {
```

Comment:
I'd prefer to just fix the backend to return the `wpcom_domain` property - otherwise we'll filter out some partner subdomains and that's not how it's working now (the partner subdomain seems to be just a regular mapped domain).

Reply:
Got it. Thanks.